dateparser/date.py

```diff
@@ -14,7 +14,7 @@
 
 
 def sanitize_spaces(html_string):
-    html_string = re.sub(u'\xa0', ' ', html_string)
+    html_string = re.sub(u'\xc2\xa0', ' ', html_string)
```

Comment:
Should there still be `u''` prefix in this case? And anyway, wouldn't this just work with `re.UNICODE` flag instead of encoding?